rp2040-hal/src/timer.rs

```diff
@@ -189,11 +188,13 @@ impl eh1_0_alpha::delay::DelayNs for Timer {
     }
 }
 
-/// Implementation of the embedded_hal::Timer traits using rp2040_hal::timer counter
+/// Implementation of the embedded_hal_0_2::Timer traits using rp2040_hal::timer counter.
+///
+/// There is no Embedded HAL 1.0 equivalent at this time.
```

Comment:
Perhaps we should note that in many cases, `DelayNs` (which is implemented by `Timer`) could be used instead?

I see two use cases where `CountDown` can't be replaced by `DelayNs`:
- you don't want to block, but do something else when `wait` returns `WouldBlock`
- you use the periodic timer (and manually doing repeated `delay_ms()` calls would be too imprecise)